protocols/virtualfund/virtualfund.go

```diff
@@ -38,17 +38,8 @@ type GuaranteeInfo struct {
 	GuaranteeDestination types.Destination
```

Comment:
There are two parts of the diffs in this file:
- the old ledger channel is removed from the Connection struct: https://github.com/statechannels/go-nitro/pull/513/commits/d0e34c5759bd0dfb4f7ff8d4178fbe4a1a669c05. Other changes made in that commit are more or less forced.
- subsequent changes, which are more reasonable: https://github.com/statechannels/go-nitro/compare/d0e34c5759bd0dfb4f7ff8d4178fbe4a1a669c05...528b1cda71736ccf32afd825307d94eb70c5f9f6#diff-e78ba83d3714e679fa001e0cc48267b733ce93be39eb4ab1a37a777d25f6ca57